### PR TITLE
feat(repo): Standardize package naming and terminology, improve overview of repo components

### LINKED ISSUE #80: Consider renaming packages to disambiguate from source programs
Eg, it might be better to call `@eternagame/vite` `@eternagame/vite-config`

### PR DESCRIPTION
**Pull request by luxaritas**
Resolves #80

### CODE REVIEW

**Comment by luxaritas** [comment]
CI fix in #87

File: README.md [hunk 1]
```diff
@@ -8,12 +8,22 @@ Interested in development? Join the discussion on the Eterna Discord!
...
+
+This will also set you up to use our nx plugin, [`@eternagame/nx`](./packages/nx), which has our standard nx configuration
+and package generators.
+
+### Configuration
+
+- [`@eternagame/tsconfig`](./packages/tsconfig) - Our standard `tsconfig`s
+- [`@eternagame/eslint-plugin`](./packages/eslint-plugin) - Our standard ESLint configuration
+- [`@eternagame/jest`](./packages/jest) - Our standard Jest configuration
+- [`@eternagame/vite`](./packages/vite) - Our standard Vite configurations
```
Comment (tkaragianes) [inline README.md:23]: Just a clarification: some of these are plural, and some are singular. Intentional? Is it that we have one Jest configuration, but potentially multiple Vite or tsconfig configurations?

Comment (luxaritas) [reply README.md:23]: Well there are multiple tsconfigs, and vite can be configured multiple ways - though I guess the jest config can be configured multiple ways too! I guess the question here would be "does a parameterized config count as one config, or many?" Any thoughts?

Comment (tkaragianes) [reply README.md:23]: When I read standard tsconfigs, I assume there are multiple tsconfig files. So a single config file that has parameters that can be adjusted feels like one config. Whereas two config files for different environments feels like multiple configs. That's a weird delineation, but it's what I naturally think here.

**Review by tkaragianes — APPROVED** [review]

**Pull request merged**